tests/test_core.py

```diff
@@ -2030,6 +2040,10 @@ def test_stream(block_length, frame_length, hop_length, mono, offset,
     # Load the reference data.
     # We'll cast to mono here to simplify checking
 
+    # File objects have to be reset before loading
+    if hasattr(path, 'seek'):
```

Comment:
Not sure how to setup a pytest.fixture in a cleaner way than this :disappointed: 

Gladly take tips!

Reply:
That seems good to me.  It's a little gnarly, but that should be expected since we're mixing different types in the fixture.